src/Microsoft.Health.Fhir.Core/Configs/AuditConfiguration.cs

```diff
@@ -0,0 +1,12 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Fhir.Core.Configs
+{
+    public class AuditConfiguration
+    {
+        public string CustomAuditHeaderPrefix { get; set; } = "X-MS-AZUREFHIR-AUDIT-";
```

Comment:
We should have validation to make sure the prefix is not empty (E.g., I cannot supply empty string in the config file).